test/e2e/network/scale/ingress.go

```diff
@@ -18,457 +15,3 @@ limitations under the License.
 */
 
 package scale
```

Comment:
This leaves just an empty file in `test/e2e/network/scale/`. You can drop it.

Reply:
Done!